CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # grinder.dart changes
 
+## UNRELEASED
+- Auto-expand directories passed to `Analyzer.analyze()` to a list of Dart 
+  source files using the added method `findDartSourceFiles()`.
+- Modified `coerceToPathList()` to accept `Iterable` instead of `List` 
```

Comment:
We probably don't need to mention `coerceToPathList` in the release notes since it's not part of the public api.